### ISSUE TITLE
Improve text contrast so theme does not fail accessibility rules

### ISSUE THREAD

**Issue by AndrewRayCode** [poychang/github-dark-theme#264]
Most of the text is gray text on a gray background which fails [accessibility guidelines](https://webaim.org/resources/contrastchecker/). The most important place to address this is in places with large blocks of text: Readmes, wikis, and tickets.

**Comment by IamCathal** [comment]
I assume that you mean text such as my username shown in the picture below

Vanilla | Github dark theme
------------ | -------------
![Before](https://i.imgur.com/FMJ4sVg.png) | ![After](https://i.imgur.com/tdAiJHZ.png)  |

The title text on this PR passes for the most part and so does the text of the comments.
![](https://i.imgur.com/zi26PXD.png)

The slightly muted text (like the sub text under Assignees, Labels, Projects etc) on this issue do fail across the board which is an issue. 

It would be a small but effective change to the project and I don't see why a PR would be straight up denied so if you're interested in doing it then go on right ahead and if not then I'm sure I can find some time to do it :)

**Comment by ErwanAliasr1** [comment]
It would be cool to have a parameter to select the text color. It's too close from the background color making the read difficult. The contrast isn't enough.

**Comment by laurent22** [comment]
Yes, the contrast is not right, for example in the comment boxes. It fails AAA and barely passes AA. Moreover, the WCAG ratings are for what they call "normal text", which is 18px, while the text on GitHub is 14px. So I believe the theme actually fails WCAG AA too at this pixel size.

To be honest, if you don't have a high DPI, high quality screen, it's easy to see that the text is hard to read.
